### PR TITLE
21.7 fb protocol title length

### PR DESCRIPTION
**Pull request by jonesgaohsu**
#### Rationale
Revision of misnamed feature branch previously submitted 
This FB changes the setScale value for the Protocol Title to 200 to match NIH TItle Length

#### Related Pull Requests
* 

#### Changes
Update of ONPRC_EHRCustomeriser.java line 1555

### CODE REVIEW

**Comment by jonesgaohsu** [comment]
Removed bad naming and created new version to correct the naming error

File: onprc_ehr/src/org/labkey/onprc_ehr/table/ONPRC_EHRCustomizer.java [hunk 1]
```diff
@@ -1545,13 +1545,14 @@ private void customizeRequests(AbstractTableInfo ti)
...
+//Update by jonesga 2021-10-01
+    //update to item below to change the setScale to 200 for protocol tritle
     private void customizeProtocol(AbstractTableInfo ti)
     {
         ti.getMutableColumn("inves").setHidden(true);
         ti.getMutableColumn("maxAnimals").setHidden(true);
-        ti.getMutableColumn("title").setScale(80);
-        ti.getMutableColumn("investigatorId").setHidden(false);
+        ti.getMutableColumn("title").setScale(200);
+        Update of ONPRC_EHRCustomizer to allow 200 characters for Title of Protocol        ti.getMutableColumn("investigatorId").setHidden(false);
```
Comment (labkey-jeckels) [inline onprc_ehr/src/org/labkey/onprc_ehr/table/ONPRC_EHRCustomizer.java]: I'm sure this is intended to be a comment but that's now how it's formatted and it's breaking the build. Please correct.

**Review by labkey-jeckels — CHANGES_REQUESTED** [review]
See my comment about the mangled comment line.

Also, please follow the agreed-upon process of having another ONPRC developer review prior to looping me in. Thanks!

**Review by jonesgaohsu — COMMENTED** [review]
Changes made

File: onprc_ehr/src/org/labkey/onprc_ehr/table/ONPRC_EHRCustomizer.java (hunk 1 above)
Comment (jonesgaohsu) [reply onprc_ehr/src/org/labkey/onprc_ehr/table/ONPRC_EHRCustomizer.java]: CHanges made and submitted to Lakshmi for review - Thanks

**Comment by jonesgaohsu** [comment]
Josh, For some reason I cannot add Raymond as a Reviewer, How can he be added?

**Comment by labkey-jeckels** [comment]
@jonesgaohsu Keith and I can add him without problems. Are you using his GitHub user name, ohsudev? He hasn't attached his real name to the account so you need to use the user name directly.

**Comment by jonesgaohsu** [comment]
With Keith's help got it added for Raymond to review

**Review by Ohsudev — APPROVED** [review]

**Review by labkey-jeckels — APPROVED** [review]

**Pull request merged**